mystery-deckbuilder/Assets/Scripts/Encounter/EncounterScript.cs

```diff
@@ -0,0 +1,68 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+/**
+ * script for the encounter gameobject
+ * to use:
+ *      encounter = Instantiate(encounter);
+        encounter.GetComponent<EncounterScript>().StartEncounter(int complianceThreshold, int startingCompliance, int startingPatience)
+ */
+public class EncounterScript : MonoBehaviour
+{
+    [SerializeField] private int threshold;
+    [SerializeField] private int compliance;
+    [SerializeField] private int patience;
+    private ArrayList deck;
```

Comment:
NIT: as per style private fields should be named _threshold, _compliance, etc. Minor enough issue to not require change.